fix(legal): update privacy policy to March 2026 (#8104)
safe-global/safe-wallet-monorepo: apps/web/src/markdown/privacy/privacy.md

   1. public Wallet address, and  
   2. WalletConnect connection data.

2. We also allow you to log-in and to the Safe{Wallet} app via Single-Sign On. For this purpose we process:

>    1. Email address (not stored by SafeLabs. See Section 5 for more details), and  

Reviewer: ```suggestion
   1. Email address (not stored by Safe Labs. See Section 5 for more details), and  
```

   2. User ID linked to the email address.

3. We process personal data when you fill out forms to register for a demo or request more information about new product integrations. Personal data processed include:

   1. Full name;
   2. Email address;
   3. Company name;
   4. Location;
   5. Responses to open text fields;

>    6. SafeWallet address (optional);

Reviewer: ```suggestion
   6. Safe{Wallet} address (optional);
```

[**WalletConnect**](https://walletconnect.com/) is used to connect wallets to dapps using end-to-end encryption by scanning a QR code. We do not store any information collected by WalletConnect.

## 5.8. Sentry {#sentry}

> We use [**Sentry**](https://sentry.io/) to collect error reports and crashes to improve our services and user experience.

Reviewer 2: @devpramoth the comment is valid, we no longer use sentry.

We do not store any information collected by Beamer.

## 5.10. Node providers {#node-providers}

We use [**Infura**](https://www.infura.io/) and [**Nodereal**](https://nodereal.io/) to query public blockchain data from our backend services. All Safes are monitored, no personalization is happening and no user IP addresses are forwarded. Personal data processed are:

* your smart contract address of the Safe,  
* transaction id/hash, and  
* Transaction data.

## 5.11. Tenderly {#tenderly}

We use [**Tenderly**](https://tenderly.co/) to simulate blockchain transactions before they are executed. For that we send your smart contract address of your Safe Account and transaction data to Tenderly.

**Internal communication**

We use the following tools for internal communication.

* [**Slack**](https://slack.com/)  
* [**Google Workspace**](https://workspace.google.com/)  
* [**Notion**](https://notion.so/)

## 5.12. Fingerprint {#fingerprint}

This tool enables the processing in sections 3.5.7 and 3.5.8.

## 5.13. Auth0 by Okta {#auth0-by-okta}

> This tool enables the login via Single Sign-On as described in section (3.1.2). Email addresses are not stored by SafeLabs but by Auth0. SafeLabs only processes the User ID generated by Auth0 tied to the user email.

Reviewer: ```suggestion
This tool enables the login via Single Sign-On as described in section (3.1.2). Email addresses are not stored by Safe Labs but by Auth0. SafeLabs only processes the User ID generated by Auth0 tied to the user email.
```
